src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,14 @@
+package org.example;
+
+import org.example.inspector.SongInspectorImpl;
+
+public class Main {
+  public static void main(String[] args) {
+    SongInspectorImpl songInspectorImpl = new SongInspectorImpl();
+    songInspectorImpl.inspect(SongInspectorImpl.getCurrentDirectory());
+  }
+}
+
```

Comment:
Shouldn't be more than one empty line in a row. Blank line at file's end is a good tradition too (but not 4).